pipeline.py

```diff
@@ -51,19 +62,33 @@ class CorpusManager:
     """
 
     def __init__(self, path_to_raw_txt_data: str):
-        pass
+        self.path = Path(path_to_raw_txt_data)
+        self._storage = {}
+        self._scan_dataset()
 
     def _scan_dataset(self):
         """
         Register each dataset entry
         """
-        pass
+        dataset = list(self.path.glob("*_raw.txt"))
+        dataset.sort(key=self._get_file_id)
```

Comment:
why do you need to sort them?

Reply:
I found it more comfortable to process the articles in the order in which they are located in the folder. But as I understand it can be unnecessary, so I removed it.